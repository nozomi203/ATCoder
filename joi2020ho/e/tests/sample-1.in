5 5
9 3 2 6 5
1 1 3
2 1 5
3 2 5
4 3 3
5 3 5
