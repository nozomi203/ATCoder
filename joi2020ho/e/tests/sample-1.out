21
39
33
9
27
